toolset/setup/linux/prerequisites.sh

```diff
@@ -56,6 +56,11 @@ sudo sh -c "echo '*               -    nofile          65535' >> /etc/security/l
 sudo sh -c "echo '*            hard    rtprio             99' >> /etc/security/limits.conf"
 sudo sh -c "echo '*            soft    rtprio             99' >> /etc/security/limits.conf"
 
+# Create a tfb command alias for running the toolset
+# For now, this still ensures you have to be in the framework root to run it
+alias tfb="./toolset/run-tests.py"
+echo 'alias tfb="./toolset/run-tests.py"' >> ~/.bash_profile
```

Comment:
If you delete the `installs` directory (which I do for testing fairly regularly), then this will get rerun and append several times.

Alternatively, I **believe** you can use `/etc/profile.d` to create a file that will be executed in new shells (would require sudo, but meh).